src/methods/getAccountInfo.ts

```diff
@@ -120,11 +131,25 @@ export const getAccountInfo = async (
         internalAddresses,
         accountEmpty,
       );
+
+      // fetch data for each byron address and set account.byronAddresses
+      const byronAccountAddresses = await getAccountAddressesData(
+        externalAddressesByron,
+        internalAddressesByron,
+        accountEmpty,
```

Comment:
Dear diary , I'm thinking this could be skipped completely since we don't want to really support byron more than necessary and list of addresses is all we need to properly calculate account's inputs/outputs in a transaction.
that would save at least 40 reqs per getAccountInfo 🤔 

For non empty shelley+ account, which presumably will be the most common type of account used with getAccountInfo, we would most likely fetch byron stuff unnecessarily. Most accounts won't touch byron anymore.

Sooo I would go with bare list of byron addresses (aka list of strings) without any metadata for them.